flint.ui/src/store/modules/point.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
Please provide a good justification for disabling ESLint.

Reply:
I was getting errors while trying to send RothC config from the state to the API. Now that has been fixed and I have enabled ESLint again!